src/translations/pages/contact.ts

```diff
@@ -158,6 +158,74 @@ export const Contact = {
         },
       },
     },
+
+    feedback: {
+      title: _(
+        'Tilbakemelding billettkontroll ',
+        'Feedback on ticket control',
+        'Tilbakemelding billettkontroll ',
+      ),
+      info: _(
+        'Har du tilbakemelding på en bestemt billettkontroll, trenger vi en beskrivelse av hvor og når billettkontrollen fant sted for å kunne gå videre med saken. Hvis du har en generell tilbakemelding, kan du hoppe over de første punktene.',
+        'If you have feedback regarding a specific ticket inspection, we need a description of where and when the ticket inspection took place in order to proceed with the matter. If you have general feedback, you can skip the initial points.',
+        'Har du tilbakemelding på ein bestemt billettkontroll, treng vi ei beskriving av kor og kva tid billettkontrollen var for å kunne gå vidare med saka. Har du ei generell tilbakemelding, kan du hoppe over dei første punkta.',
+      ),
+      locationQuestion: _(
+        'Hvor var billettkontrollen?',
+        'Where was the ticket control conducted?',
+        'Kor var billettkontrollen?',
+      ),
+      transportMode: {
+        label: _('Reisemåte', 'Transport mode', 'Reisemåte'),
+        optionLabel: _(
+          'Velg reisemåte',
+          'Choose transport mode',
+          'Velg reisemåte',
+        ),
+        notDefinedFeedback: _(
+          'Reisemåte må settes!',
+          'Transport mode must be set!',
+          'Reisemåte må settes!',
```

Comment:
I don't think we need "!" here. Let's try with something like: `Velg et transportmiddel` / `Select a transport mode`. This applies to the other error messages as well